project/plugins.sbt

```diff
@@ -4,11 +4,10 @@ resolvers += Resolver.sonatypeRepo("releases") // to more quickly obtain paradox
 
 // need this to resolve http://jcenter.bintray.com/org/jenkins-ci/jenkins/1.26/
 // which is used by plugin "org.kohsuke" % "github-api" % "1.68"
-resolvers += "Bintray Jcenter" at "https://jcenter.bintray.com/"
+resolvers += Resolver.jcenterRepo
 
 // for sbt-bintray for resolving credentials from env vars
-resolvers += Resolver.url("2m-sbt-plugin-releases", url("https://dl.bintray.com/2m/sbt-plugin-releases/"))(Resolver.ivyStylePatterns)
-resolvers += Resolver.bintrayRepo("2m", "sbt-plugin-releases")
+resolvers += Resolver.bintrayIvyRepo("2m", "sbt-plugin-releases")
```

Comment:
ahh yes you're right I ran into the same thing for akka-paradox - hopefully this will be a thing of the past after sbt 1.0 :)